src/content/docs/extras.mdx

```diff
@@ -320,6 +320,215 @@ str_detect("Hello world", "ello")
 </div>
 
   
+## arrow and duckdb
+
+_Fast data storage and database_
+
+One advantage of the learning the `tidyverse` syntax is that it plugs 
+in seemlessly with databases using the 
+[dbplyr](https://dbplyr.tidyverse.org/) 
+package. The **arrow** package provides a really fast and memory efficient data 
+storage format (`.parquet`). The **duckdb** package provides a really fast and 
+memory efficient database system. For example, imagine you have a 10M row 
+dataset with hundreds of variables. You want to take group-averages of a few 
+varaibles. Arrow/duckdb will do this *astonigshly* fast by not having to load 
+an entire CSV into memory.
+
+For this example, we will create a small dataset using the following code:
+```r
+data(airquality)
+dir.create("airquality")
+arrow::write_dataset(
+  airquality, 
+  path = "airquality",
+  partitioning = c("Month") # <-- NB: partition your data by the main grouping vars for efficiency
+)
```

Comment:
I'm totally fine running with this example. But should we maybe re-use the flights14 dataset from the main data.table section? Maybe not, but just wanted to flag.

Reply:
Agreed; good idea